p2p/server/msgserver.go

```diff
@@ -106,9 +106,10 @@ func (p *MessageServer) cleanStaleMessages() {
 			if time.Since(item.timestamp) > p.requestLifetime {
 				p.Debug("cleanStaleMessages remove request ", item.id)
 				p.removeFromPending(item.id)
-			} else {
-				return
 			}
+		} else {
```

Comment:
You can use returns instead of `else`

Reply:
after we handled a message we want to try and pull the next one out of the queue 
so we want it to exit only if the queue is empty (nil value)